cnf-certification-test/preflight/suite.go

```diff
@@ -42,6 +44,12 @@ var _ = ginkgo.Describe(common.PreflightTestKey, func() {
 		return
 	}
 
+	// Safeguard against running the preflight tests if we are specifically targeting a certain label eg. in QE.
+	ginkgoConfig, _ := ginkgo.GinkgoConfiguration()
+	if !labelsAllowTestRun(ginkgoConfig.LabelFilter, []string{common.PreflightTestKey, identifiers.TagCommon}) {
+		ginkgo.Skip("LabelFilter is set but 'preflight' or 'common' tests are not targeted.  Issuing skip.")
```

Comment:
Have you tested this? As per [ginkgo docs](https://onsi.github.io/ginkgo/#skipping-specs), Skip might not run here as this function is called uring the construction phase, not the running phase:

> You can call Skip in any subject or setup nodes. If called in a BeforeEach, Skip will skip the current spec. If called in a BeforeAll, Skip will skip all specs in the Ordered container (however, skipping an individual spec in an Ordered container does not skip subsequent specs). If called in a BeforeSuite, Skip will skip the entire suite.
>
> You cannot call Skip in a container node - Skip only applies during the Run Phase, not the Tree Construction Phase.

Reply:
You are correct!  It errored out when attempting the Skip() not inside of an It block.

I have adjusted it to just log and return.

Thanks!